vroxy.py

```diff
@@ -60,7 +61,11 @@ def extractExpiry(self) -> float:
 pool = PoolCount()
 routes = web.RouteTableDef()
 
-config = ConfigParser({"server": {"host": "localhost", "port": "8008"}})
+config = ConfigParser()
+config["server"] = {
+    "host": os.getenv("VROXY_HOST", "localhost"),
```

Comment:
```suggestion
    "host": os.getenv("VROXY_HOST", "0.0.0.0"),
```

Ehh let's just work by default for cloud stuff